src/Payments.Core/PaymentsContext.cs

```diff
@@ -8,8 +9,14 @@ public class PaymentsContext : DbContext
         public PaymentsContext(DbContextOptions<PaymentsContext> options) : base(options)
         {
             
+        }
+
+        [Obsolete("Just use for tests")]
+        public PaymentsContext()
```

Comment:
@srkirkland I added this because the other constructor was causing me grief. Any ideas to mock out the constructor with parameters would be helpful.